docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
+      resource: new LitAccessControlConditionResource("*"),
+      ability: LitAbility.AccessControlConditionDecryption,
+    },
+  ],
+  authNeededCallback: async ({
     uri,
     expiration,
     resourceAbilityRequests,
   }) => {
-    // Prepare the SIWE message for signing
-    const toSign = await createSiweMessageWithRecaps({
-      uri: uri,
-      expiration: expiration,
+    const toSign = await createSiweMessage({
+      uri,
+      expiration,
       resources: resourceAbilityRequests,
-      walletAddress: wallet.address,
+      walletAddress: await ethersSigner.getAddress(),
       nonce: await litNodeClient.getLatestBlockhash(),
-      litNodeClient: litNodeClient,
-    });
-    // Use the Ethereum wallet to sign the message, return the digital signature
-    const signature = await wallet.signMessage(toSign);
-
-    // Create an AuthSig using the derived signature, the message, and wallet address
-    const authSig = {
-      sig: signature,
-      derivedVia: "web3.eth.personal.sign",
-      signedMessage: toSign,
-      address: wallet.address,
-    };
-
-    return authSig;
-  };
-
-  // Create a session key and sign it using the authNeededCallback defined above
-  const sessionSigs = await litNodeClient.getSessionSigs({
-    chain: "ethereum",
-    resourceAbilityRequests: [
-      {
-        resource: new LitActionResource("*"),
-        ability: LitAbility.LitActionExecution,
-      },
-    ],
-    authNeededCallback,
+      litNodeClient,
     });
 
-  return sessionSigs;
-};
+    return await generateAuthSig({
+      signer: ethersSigner,
+      toSign,
+    });
+  },
+});
 ```
 
-**Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
-:::note
- If running the SDK in a Server environment, session signatures may *not* be cached unless you provide an instance of `Storage` to the runtime.
- [Here](https://www.npmjs.com/package/node-localstorage) is an implementation of `LocalStorage` which creates local files to persist storage data.
- If storage is not available, session keys *MUST* be persisted in an external data store. 
- ```javascript
- const LocalStorage = require('node-localstorage').LocalStorage;
- const litNodeClient = new LitNodeClient({
-  litNetwork: "datil-dev",
-  debug: true,
-  storageProvider: {
-    provider: new LocalStorage('./storage.test.db'),
-  }
- });
- ```
-:::
-
-The `getSessionSigs()` function will try to create a session key for you and store it in local storage. You can also generate the session key yourself using `generateSessionKeyPair()` function and store it however you like. You can then pass the generated session key to `getSessionSigs()` as the `sessionKey` param.
-
-In the example above, we construct a SIWE message manually, but you can use the `checkAndSignAuthMessage` when in a browser context.
+**Note:** The nonce should be the latest Ethereum blockhash returned by the nodes during the handshake.
 
 ## Resources You Can Request
```

Comment:
I feel like this section can be removed, and we should cover it in the `Session Signatures` overview page, or in the specific page `Lit Resources and Abilities`

I'd actually like to redo the `Lit Resources and Abilities` and `Capability Objects` pages, but that's out of scope for this PR

Reply:
I'll add this to the overview

Reply:
Will say this in the overview. I do think the `Lit Resources and Abilities` page could be redone, maybe I get to it after wrapping up session sigs?

Reply:
I've created [a ticket](https://linear.app/litprotocol/issue/DREL-69/refactor-the-lit-resources-and-abilities-docs-page) for it, but we should discuss priority in the next stand up